### DISCUSSION TITLE
SetIntStr and DictIntStrAny disappeared from encoders

### DISCUSSION THREAD

**Discussion by ricciarellif** [fastapi/fastapi#10171] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi.encoders import SetIntStr, DictIntStrAny
def route(
    response_model_include: Optional[Union[SetIntStr, DictIntStrAny]] = None,
    response_model_exclude: Optional[Union[SetIntStr, DictIntStrAny]] = None
):
... some code ...
```

### Description

cannot import name 'SetIntStr' from 'fastapi.encoders'

### Operating System

Linux

### Operating System Details

Ubuntu 22.04

### FastAPI Version

0.100.0

### Pydantic Version

1.10.12

### Python Version

3.10

### Additional Context

The FastAPI version 0.99.0 did contain the following definition in the "encoders" module:
...
from pydantic.json import ENCODERS_BY_TYPE

SetIntStr = Set[Union[int, str]]
DictIntStrAny = Dict[Union[int, str], Any]

def generate_encoders_by_class_tuples(
    type_encoder_map: Dict[Any, Callable[[Any], Any]]
...

But from the version 0.100.0 and following, both SetIntStr and DictIntStrAny disappeared.

I've looked inside the FastAPI's changelogs but didn't find any reference about this. 

There are some projects out there (like FastAPI-Gateway and similar projects) that makes use of them...

Are there any chanches that you reimplement these in the future versions?

Thank you in advance

**Comment by lifengmds** [comment]
I had the same issue. 
from fastapi.encoders import DictIntStrAny, SetIntStr
ImportError: cannot import name 'DictIntStrAny' from 'fastapi.encoders'

**Reply by kj3moraes** [reply]
I have the same issue right now.

**Comment by dipanjal** [comment]
Facing the same issue with FastAPI `0.110.0`
However, I have solved it like this 
```python
SetIntStr = Set[Union[int, str]]
DictIntStrAny = Dict[Union[int, str], Any]
```

**Comment by YuriiMotov** [comment, marked as answer]
Use `from fastapi.types import IncEx`:

```python
from fastapi.types import IncEx
def route(
    response_model_include: Optional[IncEx] = None,
    response_model_exclude: Optional[IncEx] = None
):
```
